site/dat/docs/Appium.md

```diff
@@ -0,0 +1,26 @@
+# Appium Executor
+
+## About
+[Appium](http://appium.io) is a tool for testing naitive mobile applications.
+Taurus supports only python scripts for appium. Additionally, you can use taurus services to run [Appium server](Services.md#Appium-Loader) and [Android emulator](Services.md#Android-Emulator-Loader).
```

Comment:
We support also Java/JS/Ruby. Because appium is service, not executor type. We execute some engines and only help with auto-starting appium.